### ISSUE TITLE
Invalid schema threshold for AWS::SQS::Queue property MaximumMessageSize

### ISSUE THREAD

**Issue by lukaszczechowski** [aws-cloudformation/cfn-lint#4241]
### CloudFormation Lint Version

1.39.1

### What operating system are you using?

Mac

### Describe the bug

Running the validation for attached template produces
```
E3034 500111 is greater than the maximum of 262144
test.yaml:11:7
```
The error refers to `MaximumMessageSize` property which according to the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-sqs-queue.html) can be an integer up to `1048576`

### Expected behavior

There should be no `E3034` reported

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Resources:
  TestQueue:
    Type: AWS::SQS::Queue
    DeletionPolicy: Retain
    Properties:
      SqsManagedSseEnabled: true
      ReceiveMessageWaitTimeSeconds: 0
      DelaySeconds: 300
      MaximumMessageSize: 1048576
```
